### PR TITLE
Updates to  readme.md

### PR DESCRIPTION
**Pull request by la-senda-antigua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
\ No newline at end of file
+# La Senda Antigua Web Portal
+
+In this repository we have code for the church's website.
+
+## Developers
+
+### Current developers for 2024
+
+- Hugo Quinonez
```
Comment (hugoqui) [inline README.md:9]: Quiñónez

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+- David Chachon
+- Samuel Chacon
+- Jeremiah Eguizabal
+- Josue Bautista
+- Elias Quinonez
```
Comment (hugoqui) [inline README.md:14]: Quiñónez

**Review by hugoqui — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+# La Senda Antigua Web Portal
+
+In this repository we have code for the church's website.
+
+## Developers
+
+### Current developers for 2024
+
+- Hugo Quinonez
+- David Chachon
```
Comment (909MrJerbear) [inline README.md:10]: Chacon was misspelled

**Review by 909MrJerbear — CHANGES_REQUESTED** [review]

**Review by Dave9521 — APPROVED** [review]

**Review by hugoqui — APPROVED** [review]

**Review by Chacon97 — APPROVED** [review]
Very good job Mr. Quiñónez

**Review by JoshuaB-hub — APPROVED** [review]

**Review by 909MrJerbear — APPROVED** [review]

**Pull request merged**